exports/swg-gaa.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright 2021 The Subscribe with Google Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export {
+  GaaGoogleSignInButton,
+  GaaMeteringRegwall,
+  GaaUserDef,
```

Comment:
I think this is unrelated to this PR but do GaaUserDef and GoogleUserDef need to be in gaa-main.js too?

Reply:
swg-gaa.js doesn't need these defs at runtime, so it's best we leave them out of gaa-main.js